src/App.tsx

```diff
@@ -1,95 +1,77 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
+const options = [3, 5, 10, 20];
+
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [perPage, setPerPage] = useState(options[1]);
+
+  const handleChange = (e:React.ChangeEvent<HTMLSelectElement>) => {
+    setPerPage(+e.target.value);
+    setCurrentPage(1);
+  };
+
+  const getItemsPerCurrentPage
+    = ():string[] => items
+      .filter((_, i) => i >= (currentPage - 1) * perPage
+      && i < currentPage * perPage);
+
+  const itemsPerCurrentPage = getItemsPerCurrentPage();
+
+  const firstItemPerPage = (currentPage - 1) * perPage + 1;
+  const lastItemPerPage = currentPage * perPage < items.length
+    ? currentPage * perPage
+    : items.length;
+
   return (
+
     <div className="container">
       <h1>Items with Pagination</h1>
 
       <p className="lead" data-cy="info">
-        Page 1 (items 1 - 5 of 42)
+        {`Page ${currentPage} (items ${
+          firstItemPerPage} - ${lastItemPerPage} of ${items.length})`}
       </p>
 
       <div className="form-group row">
         <div className="col-3 col-sm-2 col-xl-1">
+
           <select
```

Comment:
```suggestion
        <div className="col-3 col-sm-2 col-xl-1">
          <select
```
don't add blank lines between parent and child elements